packages/@coorpacademy-components/src/template/skill-detail/test/fixtures/default.js

```diff
@@ -0,0 +1,74 @@
+import cardsGrid from '../../../../organism/cards-grid/test/fixtures/catalog';
+
+const cards = cardsGrid.props;
+const allCourses = cards;
+const onGoingCourses = cards.list;
+
+export const defaultProps = {
+  title: 'Conflict management',
+  description:
+    'Conflict management is the skill of handling disagreements constructively through effective communication, active listening, and finding mutually beneficial resolutions. It helps navigate tensions, foster collaboration, and maintain a positive environment by preventing conflicts from escalating.',
+  metrics: {
```

Comment:
Regarding fixtures, I think we need a:
- no metrics
- huge number of courses :100:

also some without filters might be interesting

(side note: maybe swap some `news` in the card grid to false)

Reply:
I added the no metrics, but didn't add fixtures with huge numbers of courses cause the load more is handled by mooc, so the fixtures might cause misleading about actual behavior